docs/manuals/source/TasksAndConcepts/Plugins/StorageDaemonPlugins/ScsicryptoSd.rst.inc

```diff
@@ -81,40 +81,106 @@ The following additional security is needed for the following operating systems:
 Linux (SG_IO ioctl interface):
 
 
-The user running the storage daemon needs the following additional capabilities: :index:`\ <single: Platform; Linux; Privileges>`\
+.. index::
+  single: Platform; Linux; Privileges
+
+To perform the operations required for **scsicrypto**, the programs must either run as user **root** or the additional capability :strong:`CAP_SYS_RAWIO+EP` must be set.
+The |sd| normally runs as user **bareos**. Running it as **root** is not recommended.
+
+If :command:`bareos-sd` does not have the appropriate capabilities, all other tape operations may still work correctly, but you will get "Unable to perform SG\_IO ioctl" errors.
+
+-  :strong:`CAP_SYS_RAWIO+EP` (see capabilities(7))
```

Comment:
These 2 items do look lost here. Either remove the item, or prepend it with an explaining sentence.

Reply:
I propose to change the paragraph like this 
```rst
.. index::
  single: Platform; Linux; Privileges

To perform the operations required for **scsicrypto**, the programs must either run as user **root** or the additional capability :strong:`CAP_SYS_RAWIO+EP` (see capabilities(7)) must be set.
The |sd| normally runs as user **bareos**. Running it as **root** is not recommended.

If :command:`bareos-sd` does not have the appropriate capabilities, all other tape operations may still work correctly, but you will get "Unable to perform SG\_IO ioctl" errors.

For systemd you can add the following line to the bareos-sd.service: :strong:`AmbientCapabilities=CAP_SYS_RAWIO`.

To check status of capabilities of the running daemon you can use the :command:`getpcaps` followed by the pid of bareos-sd.

.. warning::

   As of systemd version 249 There's no mechanism to pass restricted flag (+ep), so the result will always be full CAP_SYS_RAWIO (eip)

```

Reply:
in last commit.